plugins/omar-wms-plugin/grails-app/services/omar/wms/WebMappingService.groovy

```diff
@@ -314,7 +318,6 @@ class WebMappingService implements InitializingBean
         double scaleY = wmsParams.height.toDouble()/(bbox.maxY-bbox.minY)
         bboxMidpoint = [y: (bbox.minY + bbox.maxY) / 2, x: (bbox.minX + bbox.maxX) / 2]
```

Comment:
why does bboxMidpoint here look like {y: blah, x: blah} but below it is {lat: blah, lon: blah} ? won't that mess up the parsing of the logs as far as location goes, @arocnies ?

Reply:
also, @tculp , I recognize this is not code you added. just wondering if it is something we should change